### PR TITLE
Apache exporter integration

### PR DESCRIPTION
**Pull request by v-zhuravlev**
This embeds https://github.com/Lusitaniae/apache_exporter

sample configs:

v1
```
metrics:
  wal_directory: /tmp/wal
  global:
    remote_write:
    - url: http://cortex.k3d.localhost:9999/api/prom/push
  configs:
    - name: default
integrations:
  apache_http:
     enabled: true
     scrape_uri: 'http://localhost:9117/server-status?auto'

```

v2
```
metrics:
  wal_directory: /tmp/wal
  global:
    remote_write:
    - url: http://cortex.k3d.localhost:9999/api/prom/push
  configs:
    - name: default
integrations:
  apache_http_configs:
    - instance: abc
      scrape_uri: 'http://localhost:9117/server-status?auto'
    - instance: xyz
      scrape_uri: 'http://localhost:9117/server-status?auto'

```

### CODE REVIEW

**Comment by v-zhuravlev** [comment]
> Looks good, can you add a `CHANGELOG.md` entry please?

done, thank you

File: pkg/integrations/v2/apache_exporter/apache_exporter.go
Comment (tpaschalis) [inline pkg/integrations/v2/apache_exporter/apache_exporter.go:55]: The v1 and v2 names should match (that's why CI is complaining)

File: pkg/integrations/v2/apache_exporter/apache_exporter.go
Comment (tpaschalis) [inline pkg/integrations/v2/apache_exporter/apache_exporter.go:108]: Nitpicky comment, makes no difference but wdyt about moving init() to the top, for consistency with other integrations?

File: pkg/integrations/v2/apache_exporter/apache_exporter.go
Comment (v-zhuravlev) [reply pkg/integrations/v2/apache_exporter/apache_exporter.go]: Ok, I decided to change name apache_http, as we tend to avoid _exporter suffixes, while just apache is too generic.

File: pkg/integrations/v2/apache_exporter/apache_exporter.go
Comment (v-zhuravlev) [reply pkg/integrations/v2/apache_exporter/apache_exporter.go]: Some other integrations have it at the bottom as it seems.

**Comment by v-zhuravlev** [comment]
> Looks good, can you add a `CHANGELOG.md` entry please?

added, thank you :)

**Review by tpaschalis — APPROVED** [review]
LGTM (minus these nits) after a first pass, nice work; let's wait for rest of the team to get some 👀  on it, too!

**Review by rgeyer — APPROVED** [review]
LGTM, thanks for adding the changelog entry. As usual, that's going to require a last minute conflict resolution before you can merge 😈

File: pkg/integrations/apache_http/apache_http.go
Comment (v-zhuravlev) [reply pkg/integrations/apache_http/apache_http.go]: Reverted back to exportable :)

Comment (tpaschalis) [reply pkg/integrations/apache_http/apache_http.go]: Sorry man, it also needed a Docstring comment (since it's exported).

Comment (v-zhuravlev) [reply pkg/integrations/apache_http/apache_http.go]: Indeed :)

**Review by mattdurham — APPROVED** [review]
LGTM

**Pull request merged**
